### DISCUSSION TITLE
iOS records with no sound and low bitrate sometimes.

### DISCUSSION THREAD

**Discussion by dinithiviranga** [llfbandit/record#125] [Q&A]
have tried to record an audio using the Record plugin(3.0.0) and there was an issue, where the recorded files (.aac) of some users didn't have any sound. The bitrate of those files were very low (4478). I haven't set any bitrate from the code, so it should be the default (128000) by the plugin.

And also, the reported issue were from iphone users who were using above iOS 15.6.1.

Any thoughts on what could go wrong here?
```
if (await _audioRecorder.hasPermission()) {
    await _audioRecorder.start(encoder: AudioEncoder.AAC, path: audioPath);
    bool isRecording = await _audioRecorder.isRecording();
    setState(() {
      if (isRecording) {
        _isRecording = true;
      }
      _recordDuration = 0;
    });
    _startTimer();
  }
} 
```

**Comment by llfbandit** [comment]
As you can imagine, mostly, this plugin collects recording parameters and sets them to underlying platforms...
Did you try with another codec?

I don't think about any evident reason to justify this... iOS SDK bug ? Users recording with bluetooth (available on iOS with automatic switching) ?
